src/Features/Blockcore.Features.Wallet/UI/Pages/Accounts.razor

```diff
@@ -0,0 +1,302 @@
+                                                <td class="align-middle">@accountBalance.AmountUnconfirmed</td>
+                                                <td class="text-center align-middle">
+                                                    <button class="btn btn-sm btn-secondary" @onclick="() => { NavigateToWallet(walletname, _account.Name); }"> <span class="oi oi-arrow-right" aria-hidden="true"></span></button>
+                                                </td>
+                                            </tr>
+                                        }
+                                    }
+                                </tbody>
+                            </table>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    }
+}
+@code
+{
+    [Parameter]
+    public string walletname { get; set; }
+
+
+
+    private void NavigateToWallet()
+    {
+        NavigationManager.NavigateTo("Wallets/");
+    }
+
+
+    private void NavigateToWallet(string walletName, string accountname)
+    {
+        NavigationManager.NavigateTo("walletview/" + walletName + "/" + accountname);
+    }
+
+    private void NavigateToWalletCreate()
+    {
+        NavigationManager.NavigateTo("walletcreate");
+    }
+    private void NavigateToWalletRecover()
+    {
+        NavigationManager.NavigateTo("walletrecover");
+    }
+    private void Resync()
+    {
+        this.WalletSyncManager.SyncFromHeight(1);
+    }
+    private async Task Rebroadcast()
+    {
+        foreach (string walletName in this.WalletManager.GetWalletsNames())
+        {
+            foreach (var unspentOutput in this.WalletManager.GetSpendableTransactionsInWallet(walletName, confirmations: 0))
+            {
+                if (unspentOutput.Confirmations == 0)
+                {
+                    var sent = await this.BroadcasterManager.BroadcastTransactionAsync(unspentOutput.Transaction.Id);
+
+                    if (!sent)
+                    {
+                        if (!string.IsNullOrEmpty(unspentOutput.Transaction.Hex))
+                        {
+                            var trx = this.Network.Consensus.ConsensusFactory.CreateTransaction(unspentOutput.Transaction.Hex);
+                            this.BroadcasterManager.BroadcastTransactionAsync(trx).GetAwaiter().GetResult();
+                        }
+                    }
+                    else
+                    {
+                        var transaction = await this.PooledTransaction.GetTransaction(unspentOutput.Transaction.Id);
+                        this.BroadcasterManager.BroadcastTransactionAsync(transaction).GetAwaiter().GetResult();
+                    }
+                }
+            }
+        }
+    }
+    private Dictionary<string, (Types.AccountBalance AccountBalance, string AccountName)> ReadAllWallets()
```

Comment:
some methods here are not used just delete them

Reply:
Done